python/ray/tests/test_memstat.py

```diff
@@ -0,0 +1,195 @@
+import ray
+import numpy as np
+import time
+from ray.internal.internal_api import memory
+
+# Unique strings.
+DRIVER_PID = "driver pid"
+WORKER_PID = "worker pid"
+UNKNOWN_SIZE = " ? "
+
+# Reference status.
```

Comment:
Can we define these constants in one place and reference them here and where we generate the summary?

Reply:
I don't think it's worth it due to the cross language problems. For tests, it is better to have redundancy for readability anyways.